src/lexer.coffee

```diff
@@ -900,7 +907,7 @@ CODE       = /^[-=]>/
 
 MULTI_DENT = /^(?:\n[^\n\S]*)+/
 
-JSTOKEN    = /^`[^\\`]*(?:\\.[^\\`]*)*`/
+JSTOKEN    = /^```([\s\S]*?)```|^`[^\\`]*(?:\\.[^\\`]*)*`/
```

Comment:
Per my last comment, I’m suggesting we convert `` \` `` to `` ` `` wherever `` \` `` appears inside a single- or triple-backtick block. That should cover this case, yes?